StudentsMVC/src/main/java/com/musala/simple/students/db/input/Input.java

```diff
@@ -0,0 +1,38 @@
+package com.musala.simple.students.db.input;
+
+import org.springframework.stereotype.Component;
+
+import com.musala.simple.students.db.dto.StudentDTO;
+
+@Component
+public class Input {
```

Comment:
The class name `Input` is not explanatory enough. Please rename the class so that it bears a name denoting the clas usage - DbInput would be better for example :)